server/views/pages/adjudicationForReport/punishmentsTab.njk

```diff
@@ -75,4 +75,20 @@
     </div>
   {% endif %}
 
+  <h2 class="govuk-heading-m govuk-!-margin-top-2 govuk-!-margin-bottom-2">Punishment comments</h2>
+
+  <br>
+  <hr>
+  <br>
+
```

Comment:
Instead of using the <br> tags here, you could include a class on the <hr> which adds some margin above and below it e.g. class="govuk-!-margin-top-2 govuk-!-margin-bottom-2"

Reply:
Done.